src/popup/components/Modals/TransferSend.vue

```diff
@@ -34,8 +34,12 @@
       <BtnMain
         class="button-action-primary"
         :disabled="error || !isConnected || !transferData.address || !transferData.amount"
-        :icon="showSendButton ? ArrowSendIcon : null"
-        :text="showSendButton ? $t('pages.send.send') : $t('modals.send.next')"
+        :icon="(showSendButton && !isMultisig) ? ArrowSendIcon : null"
+        :text="showSendButton
+          ? isMultisig
+            ? $t('modals.multisigTxProposal.proposeAndApprove')
+            : $t('pages.send.send')
+          : $t('modals.send.next')"
```

Comment:
Double ternary operator should be avoided, but if you are in need you should wrap it in a computed property

Reply:
Or maybe you could use two separate components for it

Reply:
I created computed property for this. Please check if it's OK.